content/patterns/multicloud-gitops-amx-rhoai/mcg-amx-rhoai-getting-started.adoc

```diff
@@ -0,0 +1,208 @@
++
+[source,terminal]
+----
+git checkout -b my-branch
+----
++
+[source,terminal]
+----
+vi values-global.yaml
+----
++
+[source,terminal]
+----
+git add values-global.yaml
+----
++
+[source,terminal]
+----
+git commit values-global.yaml
+----
++
+[source,terminal]
+----
+git push origin my-branch
+----
+
+. Deploy the pattern by running `./pattern.sh make install` or by using the link:/infrastructure/using-validated-pattern-operator/[Validated Patterns Operator] - both methods are described below.
+
+[id="deploying-cluster-using-patternsh-file"]
+=== Deploying the cluster by using the pattern.sh file
+
+To deploy the cluster by using the `pattern.sh` file, complete the following steps:
+
+. Login to your cluster by running the following command:
++
+[source,terminal]
+----
+oc login
+----
++
+Optional: Set the `KUBECONFIG` variable for the `kubeconfig` file path:
++
+[source,terminal]
+----
+export KUBECONFIG=~/<path_to_kubeconfig>
+----
+
+. Deploy the pattern to your cluster. Run the following command:
++
+[source,terminal]
+----
+./pattern.sh make install
+----
+
+Verify that the Operators have been installed.
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *multicluster engine for Kubernetes* 
+* *Red Hat Openshift GitOps*
+* *Validated Patterns Operator*
+* *OpenVINO Toolkit Operator*
+* *Red Hat Openshift AI*
++
+.List of Installed Operators for AMX accelerated Multicloud GitOps with Openshift AI pattern
+image::multicloud-gitops-amx-rhoai/amx-rhoai-installed-operators.png[AMX accelerated Multicloud GitOps Hub with Openshift AI installed operators,role="related thumb right"]
++
+
+
+=== Deploying the cluster by using the Validated Patterns Operator
+
+To install the *Validated Patterns Operator*:
+
+. Log in to the *Openshift Container Platform* web console and select *Operators > OperatorHub*.
+
+. Search for *Validated Patterns Operator*, open it and click *Install*.
+
++
+.Install Validated Patterns Operator step 1
+image::multicloud-gitops-amx-rhoai/amx-rhoai-validated-pattern-operator-1.png[Install Validated Patterns Operator,scale=50]
++
+
+. Choose default settings for the installation mode, namespaces and update strategy and confirm it by clicking *Install*.
+
++
+.Install Validated Patterns Operator step 2
+image::multicloud-gitops-amx-rhoai/amx-rhoai-validated-pattern-operator-2.png[Install Validated Patterns Operator,scale=50]
++
+
+. Select *Operators > Installed Operators*.
+
+. Ensure that *Validated Patterns Operator* is listed in the `openshift-operators` project with a status `Succeeded`.
+
+== Create {amx-rhoai-mcg-pattern}
+
+After succeeded installation open *Validated Patterns Operator*, go to *Pattern* tab and click *Create Pattern*.
```

Comment:
One more change request here:
After a successful installation, open the Validated Patterns Operator page. Next, go to the Pattern tab and click Create Pattern.

Reply:
done